### PR TITLE
Refactored logs in js, added api route to logs

### PR DESCRIPTION
**Pull request by mvkv**

### CODE REVIEW

File: static/js/logs.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const API_PREFIX = 'api';
+const baseUrl = window.location.origin;
+
+const renderLogs = (logs) => {
+    const logsNode = document.getElementById('logs');
+    const root = document.createElement('div')
+    logs.forEach( (el) => {
+        const pNode = document.createElement('p');
+        pNode.innerText = el
```
Comment (powahftw) [inline static/js/logs.js]: nit: You could use a template literals string to do it in a single statement

File: static/js/logs.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+const getLogs = async () => {
+    const requestUrl = `${baseUrl}/${API_PREFIX}/logs`;
+    const responseData = await (await fetch(requestUrl)).json();
+    renderLogs(responseData['logs']);
+    console.log(responseData)
+}
+
+window.onload = () => {
+    getLogs();
+};
```
Comment (powahftw) [inline static/js/logs.js]: newline

File: static/js/logs.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        root.appendChild(pNode)
+    })
+    logsNode.appendChild(root)
+}
+
+const getLogs = async () => {
+    const requestUrl = `${baseUrl}/${API_PREFIX}/logs`;
+    const responseData = await (await fetch(requestUrl)).json();
+    renderLogs(responseData['logs']);
+    console.log(responseData)
```
Comment (powahftw) [inline static/js/logs.js]: ?

**Review by powahftw — CHANGES_REQUESTED** [review]

File: static/js/logs.js (hunk 3 above)
Comment (mvkv) [reply static/js/logs.js]: Removed

File: static/js/logs.js (hunk 2 above)
Comment (mvkv) [reply static/js/logs.js]: Done

File: static/js/logs.js (hunk 1 above)
Comment (mvkv) [reply static/js/logs.js]: I don't think it's possible, because appendChild needs to have a Node argument.

**Review by powahftw — APPROVED** [review]

**Pull request merged**